tests/utils/deployments.go

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import (
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+)
+
+func WaitForDeploymentReplicasReadyOrDie(c *kubernetes.Clientset, namespace, name string) {
+	if err := WaitForDeploymentReplicasReady(c, namespace, name); err != nil {
+		glog.Fatal(errors.Wrapf(err, "Failed waiting for deployment \"%s/%s\" replicas to become Ready", namespace, name))
+	}
+}
+
+func WaitForDeploymentReplicasReady(c *kubernetes.Clientset, namespace, name string) error {
+	return wait.PollImmediate(2*time.Second, 3*time.Minute, func() (done bool, err error) {
```

Comment:
Might want to define some constants on the polling timeouts, like the other utility functions.